docs/concepts/plugins.md

```diff
@@ -142,17 +142,47 @@ class Deploy {
 module.exports = Deploy;
 ```
 
+### Serverless instance
+
+The serverless instance which enables access to the whole Serverless setup during runtime is passed as the first
+parameter of the plugin constructor.
+
+class MyPlugin {
+  constructor(serverless) {
```

Comment:
only has one option, but it will actually receive the options as the second parameter